static/css/restyle.less

```diff
@@ -956,16 +956,77 @@ button.good {
 
   > ul.personas {
     border: 0;
+
+    li {
+      margin: 1em 0 0;
+      padding: 0;
+    }
+  }
+}
+
+.persona,
+.persona-hover {
+  border: 0;
+
+  .persona-inner {
+    background: none;
+  }
+
+  .persona-preview [data-browsertheme] {
+    border: .25em solid #c8e8f2 !important;
   }
+}
+
+.persona-install {
+  border: 0;
+  bottom: -4px;
+  left: -4px;
 
-  .persona-install {
+  .add {
+    background-image: none;
+    background-color: #57bd35;
     border: 0;
+    text-shadow: none;
+  }
 
-    .add {
-      background-image: none;
-      background-color: #57bd35;
-      text-shadow: none;
-    }
+  button {
+    border-radius: 0;
+  }
+}
+
+.persona-inner {
+  border-radius: 0;
+  padding: 0;
+}
+
+.persona .created,
+.persona .reviews,
+.persona .users {
+  color: #000;
+}
+
+.persona-hover .persona-preview {
+  background: none;
+}
+
+.category-landing .addons-column ul li {
+  overflow: inherit;
+
+  a {
+    padding: 0;
+  }
+
+  .persona-preview .created,
+  .persona-preview .users,
+  .persona-preview .reviews,
+  .persona-preview .review-decal {
+    float: left;
```

Comment:
Again, this seems to not be needed anymore, it seems to not even affect anything. I'll remove, though it seemed to work fine.